legacy-client.yml

```diff
@@ -0,0 +1,14 @@
+version: '2'
+services:
+
+    #
+    # mender-legacy-client 
+    #
+    mender-client:
+            image: mendersoftware/mender-client-qemu:1.7.0
+            networks:
+                - mender
+            stdin_open: true
+            tty: true
+            privileged: true
```

Comment:
Only specify what you need to override. The `image` alone should be enough here. See also comment in `common_setup.py`.

Reply:
Fixed!